template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Program.cs

```diff
@@ -16,7 +16,7 @@ public static void Main(string[] args)
             var host = new WebHostBuilder()
                 .UseKestrel()
                 .UseContentRoot(Directory.GetCurrentDirectory())
-                .ConfigureConfiguration((context, builder) => {
+                .ConfigureAppConfiguration((context, builder) => {
```

Comment:
The packages are currently pinned at build [24507](https://github.com/dotnet/templating/blob/rel/vs2017/post-rtw/template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Company.WebApplication1.csproj#L19)